src/Components/Landing/Landing.module.scss

```diff
@@ -0,0 +1,172 @@
+$title-color: rgba(0, 0, 0, 0.86);
+$button-background-color: #466eb6;
+$button-background-color_hover: #00CED1;
```

Comment:
Place this variables in `index.scss` under :root as other colours, first check whether the colours are already present there, as it might be present because we are following a theme. Use that variable using `var(--variable-name)`

Reply:
put them in index.scss